docs/public/secure-registry.adoc

```diff
@@ -0,0 +1,35 @@
+== Introduction to secure registry
+
+=== What is a secure registry?
+
+A secure registry is a registry that user can only use credential to access.
+
+=== Where to host secure registry?
+
+User can host secure registry on private GitHub repository, Enterprise GtiHub repository and cluster.
```

Comment:
`cluster` isn't clear what it means here.

Since odo doesn't support secured devfile registries running in a secured web server on the cluster yet, I think we should just remove that section and update it when odo does support it.

Reply:
Done